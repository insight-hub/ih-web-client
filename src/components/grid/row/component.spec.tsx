import React from 'react';
import renderer from 'react-test-renderer';

import { Row } from './';

describe('Row', () => {
  it('should render correct', () => {
    const component = renderer.create(<Row />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
