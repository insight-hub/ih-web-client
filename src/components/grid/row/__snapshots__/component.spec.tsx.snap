// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render correct 1`] = `
<div
  className="sc-bcXHqe sc-gswNZR dpNPGM cnJYyj"
/>
`;
